sigopt/cli/commands/training_run/unarchive.py

```diff
@@ -0,0 +1,19 @@
+import click
+import sys
+from sigopt.logging import print_logger
+from sigopt.factory import SigOptFactory
+from ...arguments import project_option
+from .base import training_run_command
+
+
+@training_run_command.command()
+@click.argument("RUN_ID")
+@project_option
+def unarchive(run_id, project):
+  '''unarchive a SigOpt TrainingRun.'''
+  try:
+    factory = SigOptFactory(project)
+    factory.connection.training_runs(run_id).update(deleted=False)
+  except Exception as e:
+    print_logger.error(f'Error: {e}')
+    sys.exit(-1)
```

Comment:
you should probably just make this a decorator if you want it to be applied to every command

Reply:
it should work if you use `functools.wraps` and make it the last decorator

Reply:
Good to know.